### PR TITLE
task: add kubebuilder, controller-tools, controller-runtime to prow

### PR DESCRIPTION
**Pull request by alexeldeib**
@DirectXMan12 @shawn-hurley FYI

### CODE REVIEW

File: config/jobs/kubernetes-sigs/controller-runtime/controller-runtime-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    decorate: true
+    path_alias: "sigs.k8s.io/controller-runtime"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190420-93fab49-master
+        command:
+        - "./hack/check-everything.sh"
+        env:
+        - name: TRACE
+          value: "1"
```
Comment (krzyzacy) [inline config/jobs/kubernetes-sigs/controller-runtime/controller-runtime-presubmits.yaml]: nit: new line in the end, also other files.

**Comment by krzyzacy** [comment]
/ok-to-test

can you also create OWNERS files for the new repo subdirs, and assign people from your team/sig to lgtm?

**Comment by alexeldeib** [comment]
Will do. I think we will also want to add some bits to the base images before everything all is said and done. 

/assign @DirectXMan12 (not sure who else would want to look at this, i'll defer to him)

**Comment by alexeldeib** [comment]
I need to update the base image + finish tweaks on the kubebuilder side.

ref https://github.com/kubernetes-sigs/kubebuilder/pull/689

**Comment by DirectXMan12** [comment]
/remove-lifecycle stale

I'd like to figure out how to get back to this

**Comment by alexeldeib** [comment]
@alvaroaleman I started this quite a while ago and never finished due to capacity issues. I'm happy to clean up and rebase. I wasn't quite sure how far you already got -- it looked like you merged some of the scripts for testing into the other repos, but I hadn't seen a test-infra PR containing all of them yet?

If you're making good progress and would like to finish, go ahead, I don't mind closing these. Just let me know either way (and i'd make the corresponding changes -- appreciate the pointers!) 😄

**Comment by alexeldeib** [comment]
closing in favor of https://github.com/kubernetes/test-infra/pull/14692 

will break this up and work on each chunk

**Pull request closed without merging**